### PR TITLE
Quickfix - outline draws the 'x' in the right place(s)

### PR DESCRIPTION
**Pull request by maltenuhn**
**Problem:**
The prototype parent marks used the letter 'x', which therefore wasn't perfectly center-aligned.

**Fix:**
This adds a new SVG-based control, which is (self)aligned perfectly
NB: like all of the experimental canvas controls, it's not yet zoom-capable, but is zoom-friendly because of the below v v v.

**Details**
The SVG here has a fixed width, and is self-centering because of the `translate: '-50% -50%'` at any zoom/scale factor.

### CODE REVIEW

File: editor/src/components/canvas/controls/outline-control.tsx [hunk 1]
```diff
@@ -55,6 +55,45 @@ function isDraggingToMove(
   return dragState != null && dragState?.type === 'MOVE_DRAG_STATE' && targetIsDragged
 }
 
+interface CenteredCrossSVGProps {
+  id: string
+  style?: React.CSSProperties
+}
+
+const CenteredCrossSVG = ({ id, style }: CenteredCrossSVGProps) => {
```
Comment (seanparsons) [inline editor/src/components/canvas/controls/outline-control.tsx:63]: Should be wrapped in `betterReactMemo`.

**Review by seanparsons — APPROVED** [review]

**Review by Rheeseyb — APPROVED** [review]

**Review by enidemi — APPROVED** [review]

File: editor/src/components/canvas/controls/outline-control.tsx [hunk 2]
```diff
@@ -55,6 +56,48 @@ function isDraggingToMove(
   return dragState != null && dragState?.type === 'MOVE_DRAG_STATE' && targetIsDragged
 }
 
+interface CenteredCrossSVGProps {
+  id: string
+  style?: React.CSSProperties
```
Comment (Rheeseyb) [inline editor/src/components/canvas/controls/outline-control.tsx:61]: Unless we're intending to pass in custom styling here, this should probably just be replaced with `left` and `top` props

File: editor/src/components/canvas/controls/outline-control.tsx (hunk 1 above)
Comment (maltenuhn) [reply editor/src/components/canvas/controls/outline-control.tsx]: done

File: editor/src/components/canvas/controls/outline-control.tsx (hunk 2 above)
Comment (maltenuhn) [reply editor/src/components/canvas/controls/outline-control.tsx]: done

**Pull request merged**